invenio_notifications/models.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Invenio-Notifications is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Models used for notifications."""
```

Comment:
question: should the file be called `dataclasses`? while it is a representation of the data there is no actual DB model in this module.

Reply:
Is models always referring to the db models in invenio? If so, I'll gladly move this.